sap-actions/src/automation/sap/utils/credentialAttributeMapper.ts

```diff
@@ -0,0 +1,42 @@
+interface DecryptedCredential {
+    id: string;
+    name: string;
+    order: number;
+    template: string;
+    attributes: {
+        id: string;
+        name: string;
+        value: string;
+    }[];
+}
+
+
+/**
+ * Takes array of decrypted credentials and returns mapped credential attributes in expected type.
+ *
+ * @param credentials Single element array of decrypted credentials matched for specific action
+ * @returns Mapped credential attributes to single object with expected type.
+ */
+export const credentialAttributesMapper = <R = Record<string, any>>(
+    credentials: DecryptedCredential[]
+): R | undefined => {
+    if (!credentials || !credentials.length) return;
```

Comment:
I would fix this line and add some error handling in case no credentials, can you fix this also in main repo?